src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -37,19 +45,17 @@ export const updateBrowserActivated = createStandardAction(
   'browser/update-browser-activated'
 )<boolean>();
 
-export const activateBrowser = (browserEl: HTMLDivElement) => {
+export const activateBrowser = () => {
   return (dispatch: Dispatch) => {
     const { protocol, host: currentHost } = location;
     const host = config.apiHost || currentHost;
-    const activateEvent = new CustomEvent('bpane-activate', {
-      bubbles: true,
-      detail: {
-        'config-url': `${protocol}${host}/browser/config`,
-        key: 'main'
-      }
-    });
 
-    browserEl.dispatchEvent(activateEvent);
+    const payload = {
+      'config-url': `${protocol}${host}/browser/config`,
+      key: 'main', // TODO: remove this field after we confirmed that it is redundant
```

Comment:
Is this something you wanted to confirm now or later?

Reply:
Later. @ens-ds23 wanted to get rid of the key field, since we are now passing the `selector` field in the payload; but he says it's too deeply rooted in genome browser code at the moment and will require some time to refactor out.